03-make/make_simplest/Makefile

```diff
@@ -1,9 +1,10 @@
+OBJS := $(main.o)
 
 hello: main.o
-	gcc -o hello main.o -lc
+	gcc -o hello main.o
 
-main.o: main.c
-	gcc -c main.c -o main.o
+%.o: %.c
+	gcc -c main.c
```

Comment:
Wrong rule actions. It is correct only for target main.o

Reply:
Fixed